posts/inside-rust/2022-04-13-imposter-syndrome.md

```diff
@@ -0,0 +1,91 @@
+author: Jane Lusby, Project Director of Collaboration
+team: Rust Foundation Project Directors <https://foundation.rust-lang.org/about/>
+---
+
+*Preface: This is in response to some feedback the project directors received
+from the Rust Foundation staff. Some of the contributors they'd talked to said
+they didn't feel justified in applying for Foundation grants even though they'd
+love the opportunity, because they don't feel qualified or deserving of them
+compared to the other amazing contributors they look up to within the Rust
+project. This was a little bit heart breaking to me, because I know exactly
+what that feeling is like[^1], and I also know just how wrong they are.*
+
+Imposter syndrome is an insidious problem within software communities. Many of
+us, especially members of marginalized communities, struggle to shake the
+feeling that we aren't as qualified as our peers. This makes us feel
+unqualified and undeserving compared to those around us. It can make us
+hesitate to join communities in the first place and, for those already
+involved, it can create a sense of impending doom where you constantly feel
+like you're going to get found out and expelled from the community. Overall
+it's just not great for mental health, 0/10, would not recommend.
+
+The thing is though, imposter syndrome is a logical fallacy[^2]. Imposter
+syndrome occurs when we discount what we know and inflate what we think other
+people know, and this effect is often then reinforced by systemic bias for
+those of us who don't get the assumption of competence.
+
+![picture of imposter syndrome, left side shows a large circle saying "What I think others know" and a small circle inside of it saying "What I know", right side shows the same small circle saying "What I know" surrounded by many other equally sized small circles labeled "What others know"](/static/images/2022-04-13-imposter-syndrome/imposter_syndrome.jpg)
+
+In reality, we're all specialists within the Rust project. We all have areas
+where we have deep expertise and other large areas where we only have the
+vaguest idea of how things work. Niko still comes to me to ask questions about
+error handling. I frequently need to tell my fellow contributors that I have no
+idea what the acronyms they're using mean[^3]. But this doesn't mean we don't
+deserve our positions within the project. We don't expect every contributor
+know everything, to be perfect, or to make no mistakes. The only thing we
+expect from our contributors is the ability to collaborate effectively with
+others and the ability to learn and grow over time.
+
+The thing that makes the Rust project as good as it is isn't a couple of
+prolific contributors lifting mountains by themselves, it's everyone working
+together that brought us to where we are today. We all make mistakes. The
+project has layer[^4] after layer[^5] of safeguards to make sure we have a
+chance to catch and fix them before they affect our users. These incidents are
```

Comment:
Also, even if all of these fail, we have procedures to handle such cases and fix such mistakes . Const promotion, I'm looking at you!